modules/calypsoify/mods.js

```diff
@@ -0,0 +1,42 @@
+window.onload = function( $ ) {
```

Comment:
setting `window.onload` here is probably a bad idea as it could conflict if any other plugin / theme / etc is doing the same thing.  Better to use `$( window ).load(function() {` -- but it doesn't pass jQuery as the first param as .ready does, so you'll need to do something like `var $ = jQuery` or the like

Reply:
Actually -- why is this taking $ if we're just hardcoding the full jQuery in each call down below?